r/R/trajectory_analysis.R

```diff
@@ -16,9 +16,14 @@
 #' @return a json with nodes and umap coordinates
```

Comment:
```suggestion
#' @return a json with nodes coordinates
```
I'd suggest to change also the comments above, removing any reference to umap coordinates, so that the roxygen documentation can be updated

Reply:
changed